src/plone/api/tests/test_content.py

```diff
@@ -1014,7 +1014,7 @@ def test_find_interface_dict(self):
                 "operator": "and",
             },
         )
-        self.assertEqual(len(brains), 1)
+        self.assertEqual(len(brains), 2)
```

Comment:
```suggestion
        # Plone Site also implements the IContentish and INavigationRoot interfaces.
        self.assertEqual(len(brains), 2)
```

Reply:
The assert on line 1026 also needs to be fixed:

https://github.com/plone/plone.api/blob/6504c3f277e09917627179fbacb102809679f467/src/plone/api/tests/test_content.py#L1026